src/daemon.rs

```diff
@@ -0,0 +1,32 @@
+use std::ffi::OsStr;
+use std::io;
+use std::os::unix::process::CommandExt;
+use std::process::{Command, Stdio};
+
+pub fn spawn_daemon<I, S>(program: &str, args: I) -> io::Result<()>
```

Comment:
```suggestion
pub fn spawn<I, S>(program: &str, args: I) -> io::Result<()>
```

This is referenced as `daemon::` anyway, so might as well cut out the duplicate `daemon`.

Reply:
But `daemon::spawn_daemon` sounds a bit more fun, it's like a command, but yeah, I got the point. I've just ported it from alacritty without a name adjustment.